src/components/organisms/NavBarSchedule/utils.ts

```diff
@@ -31,6 +31,22 @@ export const prepareForSchedule = (
   };
 };
 
+export const prepareForCalendar = (usersEvents: MyPersonalizedSchedule) => (
+  event: WithVenueId<VenueEvent>
+): PersonalizedVenueEvent => {
+  const startOfEventToShow = eventStartTime(event);
+  const endOfEventToShow = eventEndTime(event);
+
+  return {
+    ...event,
+    start_utc_seconds: getUnixTime(startOfEventToShow),
+    duration_minutes: differenceInMinutes(endOfEventToShow, startOfEventToShow),
+    isSaved: isTruthy(
+      event.id && usersEvents[event.venueId]?.includes(event.id)
+    ),
+  };
+};
```

Comment:
What is this function adding/doing differently (and why) compared to `prepareForSchedule` above? Is it just the differences in `startOfEventToShow` / `endOfEventToShow`?

It would be good to avoid duplicating code here.